<!-- path_follower_arashi launch
    Copyright Â© 2018 Kazuya Arashi. All rights reserved.
-->

<launch>
  <node name="abep_controller" pkg="abep_controller" type="abep_controller" output="screen">
    <param name="kp" value="3" type="double"/>
    <param name="kp_yaw" value="1" type="double"/>
    <param name="ki" value="0" type="double"/>
    <param name="ki_yaw" value="0" type="double"/>
    <param name="kd" value="5" type="double"/>
    <param name="kd_yaw" value="2" type="double"/>
    <param name="hz" value="10" type="double"/>
    <param name="vx_const" value="1" type="double"/>
    <param name="vy_const" value="1" type="double"/>
    <param name="vz_const" value="1" type="double"/>
    <param name="vyaw_const" value="1" type="double"/>
  </node>

</launch>